OpenLaTeXStudio/EditorModule/src/latexstudio/editor/EditorTopComponent.java

```diff
@@ -14,10 +14,18 @@
 import java.util.prefs.PreferenceChangeEvent;
 import java.util.prefs.PreferenceChangeListener;
 import java.util.prefs.Preferences;
+<<<<<<< HEAD
```

Comment:
There are unresolved conflicts in this file (as indicated by the Git <<<<<< and >>>>>>)